src/components/Error.tsx

```diff
@@ -0,0 +1,13 @@
+export const Error = () => {
+  return (
+    <p data-cy="peopleLoadingError">Something went wrong</p>
+  );
+};
+
+export const NoPeopleMessage = () => {
```

Comment:
pls avoid defining several components inside 1 file, move `NoPeopleMessage` to a separate file